ResearchKit/Common/ORKHealthAnswerFormat.m

```diff
@@ -154,26 +164,52 @@ - (ORKAnswerFormat *)impliedAnswerFormat {
             impliedAnswerFormat = format;
             
         } else if ([identifier isEqualToString:HKCharacteristicTypeIdentifierDateOfBirth]) {
-            NSCalendar *calendar = [NSCalendar currentCalendar];
-            NSDate *now = [NSDate date];
-            NSDate *thirtyFiveYearsAgo = [calendar dateByAddingUnit:NSCalendarUnitYear value:-35 toDate:now options:0];
-            NSDate *minDate = [calendar dateByAddingUnit:NSCalendarUnitYear value:-150 toDate:now options:0];
-            NSDate *maxDate = [calendar dateByAddingUnit:NSCalendarUnitDay value:1 toDate:now options:0];
-            
-            ORKDateAnswerFormat *format = [ORKDateAnswerFormat dateAnswerFormatWithDefaultDate:thirtyFiveYearsAgo
-                                                                           minimumDate:minDate
-                                                                           maximumDate:maxDate
-                                                                              calendar:nil];
+            ORKDateAnswerFormat *format = [ORKDateAnswerFormat dateAnswerFormatWithDefaultDate:_defaultDate
+                                                                           minimumDate:_minimumDate
+                                                                           maximumDate:_maximumDate
+                                                                              calendar:_calendar];
             impliedAnswerFormat = format;
         }
     }
     return impliedAnswerFormat;
 }
 
+- (void)setDefaultDate:(NSDate *)defaultDate {
+    if (!defaultDate) {
+        defaultDate = [_calendar dateByAddingUnit:NSCalendarUnitYear value:-35 toDate:[NSDate date] options:0];
```

Comment:
Do think default value should be generated during runtime?
In case, someone created one instance and archived, 10 years later, the system load this from archive:)